# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/llvm/hotfix-powerpc-pwr.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

POWER4 is not 970, and POWER5 is not G5.

--- llvm-20.1.8.src/lib/TargetParser/Host.cpp.vanilla	2025-08-26 18:55:39.053276546 +0200
+++ llvm-20.1.8.src/lib/TargetParser/Host.cpp	2025-08-26 18:56:02.654480500 +0200
@@ -146,11 +146,11 @@
       .Case("7447", "7400")
       .Case("7455", "7450")
       .Case("G4", "g4")
-      .Case("POWER4", "970")
+      .Case("POWER4", "pwr4")
       .Case("PPC970FX", "970")
       .Case("PPC970MP", "970")
       .Case("G5", "g5")
-      .Case("POWER5", "g5")
+      .Case("POWER5", "pwr5")
       .Case("A2", "a2")
       .Case("POWER6", "pwr6")
       .Case("POWER7", "pwr7")
